### PR TITLE
Internal improvement: Allow ES2021 globals in ESLint

### PR DESCRIPTION
**Pull request by haltman-at**
We need this so it doesn't complain about `BigInt` and such!

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
   "parser": "@typescript-eslint/parser",
   "plugins": ["@typescript-eslint"],
   "parserOptions": {
-    "ecmaVersion": 9,
+    "ecmaVersion": 2021,
```
Comment (cds-amal) [inline .eslintrc.json:13]: according to the docs, this entry may not be necessary. We could drop it, or set it to `12`.

Comment (haltman-at) [reply .eslintrc.json:13]: That's true; I just changed it for consistency, even though it's not necessary.

Comment (haltman-at) [reply .eslintrc.json:13]: OK after talkign to @cds-amal more I'm just going to delete this option for simplicity.

**Review by cds-amal — APPROVED** [review]
Thanks @haltman-at. let!s get it in!

**Pull request merged**